physics/module_sf_noahmp_glacier.F90

```diff
@@ -2646,9 +2646,9 @@ subroutine snowwater_glacier (nsnow   ,nsoil   ,imelt    ,dt     ,sfctmp , & !in
 
 !to obtain equilibrium state of snow in glacier region
        
-   if(sneqv > 2000.) then   ! 2000 mm -> maximum water depth
```

Comment:
Rather than have the same value in two places, it would be best if max. water depth defined as a parameter.